.github/workflows/release-stable.yml

```
@@ -30,13 +27,21 @@ jobs:
       RCLONE_CONFIG_SPACES_ENDPOINT: ams3.digitaloceanspaces.com
       RCLONE_CONFIG_SPACES_ACL: public-read
     steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 500
+      - name: Fetch tag annotations
+        run: |
+          # Fetch tag manually because fetch-tags option for checkout@v4 does not work
+          git fetch --force --tags --depth 1
```

The tag could not be on the first commit, but probably will be buried somewhere inside the history.
```suggestion
          git fetch --force --tags --depth 500
```